lib/providers/env_var_provider.ex

```diff
@@ -223,6 +230,16 @@ defmodule EnvVar.Provider do
     env_value
   end
 
+  def convert(env_value, :boolean) do
+    case env_value do
+      "1" -> true
+      "0" -> false
+      "true" -> true
+      "false" -> false
```

Comment:
I'm intentionally not including capitalized text here because we don't need to support it and it's ugly :)